docs/content/en/docs/data/data-source/scan_sql.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "scan_sql"
+linkTitle: "scan_sql"
+weight: 210
+superheading: "catalog_data_source."
+---
+
+
+
+``scan_sql(data_source_id: str, sql_request: ScanSqlRequest)``
+
+Analyze SELECT SQL query in the request. Return description of SQL result-set as list of column names with GoodData data types and list of example data returned by SQL.
```

Comment:
Personally I would change "in the request" to "in a given request". I would also consider not repeating the description of the return value.

Reply:
Fixed